packages/commonwealth/server/util/cosmosCache.ts

```diff
@@ -0,0 +1,114 @@
+import type { Request, RequestHandler } from 'express';
+import { cacheDecorator } from 'common-common/src/cacheDecorator';
+
+const oneBlock = 6; // typical Cosmos block time is ~6 seconds
+
+export const cosmosCacheRPC = (duration: number): RequestHandler => {
+  // RPC specific codes from cosmJS requests
+  const activeCodes = ['0801', '0802']; // ['DepositPerion', 'VotingPeriod']
+  const completedCodes = [
+    '0803', // 'Passed'
+    '0804', // 'Rejected'
+    '0805', // 'Failed'
+    '0803220a0a080000000000000087', // 'Passed'
```

Comment:
What do these codes mean specifically? Are they chain-specific, or global?

Reply:
Looked into this a bit. They seem Osmosis-specific. Not regen or juno.

Reply:
I labeled them Osmosis and left them in. No harm to other chains. Not totally sure what they are about, but we want them cached.